src/index.ts

```diff
@@ -1,47 +1 @@
-import {definePlugin} from 'sanity'
-import {ImageIcon} from '@sanity/icons'
-import type {AssetSource} from 'sanity'
-import FormBuilderTool from './components/FormBuilderTool'
-import Tool from './components/Tool'
-import mediaTag from './schemas/tag'
-import {MediaToolOptions} from '@types'
-
-const plugin = {
-  icon: ImageIcon,
-  name: 'media',
-  title: 'Media'
-}
-
-export const mediaAssetSource: AssetSource = {
-  ...plugin,
-  component: FormBuilderTool
-}
-
-export const media = definePlugin<MediaToolOptions | void>(options => ({
-  name: 'media',
-  form: {
-    file: {
-      assetSources: prev => {
-        return [...prev, mediaAssetSource]
-      }
-    },
-    image: {
-      assetSources: prev => {
-        return [...prev, mediaAssetSource]
-      }
-    }
-  },
-  schema: {
-    types: [mediaTag]
-  },
-  tools: prev => {
-    return [
-      ...prev,
-      {
-        ...plugin,
-        options,
-        component: Tool
-      }
-    ]
-  }
-}))
+export {media} from './plugin'
```

Comment:
As is, this is a breaking change (no longer exporting the asset source).
 
 You _could_ do
```suggestion
export * from './plugin'
```

But I prefer us being explicit
 ```suggestion
export {media, mediaAssetSource} from './plugin'
```

Looking at it now, we probably should export `MediaToolOptions` too, since its used in userland

 ```suggestion
export {media, mediaAssetSource} from './plugin'
export {MediaToolOptions} from '@types'
```

Reply:
Good suggestions! Added now 👍